Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -189,6 +194,7 @@ - (void)testSwizzlingNonAppDelegate {
   [invalidAppDelegate application:self.mockSharedApplication
      didReceiveRemoteNotification:@{}
            fetchCompletionHandler:^(UIBackgroundFetchResult result) {}];
+#endif
```

Comment:
I wonder, if we can keep the test for macOS if we use method `application:didReceiveRemoteNotification:` for macOS and `application:didReceiveRemoteNotification:fetchCompletionHandler:` for iOS and tvOS?

Reply:
but application:didReceiveRemoteNotification: is not available for macOS: https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623117-application?language=objc

the second is already for iOS and tvOS.